libbeat/docs/upgrading.asciidoc

```diff
@@ -61,30 +57,18 @@ other products in the {stack}, upgrade {beats} as part of the
 Upgrading to 6.7 is required because the {es} index template was modified to
 be compatible with {es} 7.0 (the `_type` setting changed from `doc` to `_doc`).
 
-After upgrading to 6.7, make sure the 6.7 template gets loaded. To load the
-template, run the `setup` command with the `--template` option specified. 
-For example, if the {beats} output is {es}, run:
+After upgrading to 6.7, make sure the 6.7 template gets loaded. 
```

Comment:
It might just be me, but I feel slightly less mental friction when I see "index template" instead of "template". When I was first learning about index templates, inconsistent references to "index templates" and "templates" made me paranoid that there was another kind of template of which I wasn't aware.

Reply:
Also, when we say "make sure it gets loaded", how do we expect the user to do that? Should we give them a command to run, or maybe just be more explicit and provide a link? For example:

> After upgrading to 6.7, use the [Index Template API](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html#getting) to verify that the 6.7 index template has been created in Elasticsearch.

Reply:
Yup, totally agree. I've made the references to index template consistent (at least in the upgrade docs) and will provide a link to the index template API as you've suggested.